Issue updater failing since 2026-03-08

## Description
The issue updater workflow started failing on 2026-03-08

with

```
java.io.UncheckedIOException: org.kohsuke.github.HttpException: {"message":"Must have admin rights to Repository.","documentation_url":"https://docs.github.com/rest/issues/issues#update-an-issue","status":"403"}
	at Report.run(quarkus-ecosystem-issue.java:159)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at Report.main(quarkus-ecosystem-issue.java:376)
Caused by: org.kohsuke.github.HttpException: {"message":"Must have admin rights to Repository.","documentation_url":"https://docs.github.com/rest/issues/issues#update-an-issue","status":"403"}
	at org.kohsuke.github.GitHubConnectorResponseErrorHandler$1.onError(GitHubConnectorResponseErrorHandler.java:88)
	at org.kohsuke.github.GitHubClient.detectKnownErrors(GitHubClient.java:504)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:464)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:427)
	at org.kohsuke.github.Requester.send(Requester.java:70)
	at org.kohsuke.github.GHIssue.edit(GHIssue.java:293)
	at org.kohsuke.github.GHIssue.reopen(GHIssue.java:343)
	at Report.reportJobResults(quarkus-ecosystem-issue.java:198)
	at Report.run(quarkus-ecosystem-issue.java:152)
	... 9 more
```

## Additional context

GitHub pushed [a REST API update on March 12](https://github.blog/changelog/2026-03-12-rest-api-version-2026-03-10-is-now-available/) but since the workflow was breaking before that it's not clear if it's related or not.

After all `github-api` still uses the old API version:

https://github.com/hub4j/github-api/blame/9152b378276fae69d66ef626e3e57d01dda2b04b/src/main/java/org/kohsuke/github/GitHubClient.java#L270

Also the update is opt-in, so even without the above we shouldn't be using the new API either way

> Version 2022-11-28 will continue to be fully supported for at least 24 months from today, and requests that don't include the X-GitHub-Api-Version header will continue to default to 2022-11-28.

--- a/.github/quarkus-ecosystem-issue.java
+++ b/.github/quarkus-ecosystem-issue.java
@@ -17,7 +17,7 @@
 //usr/bin/env jbang "$0" "$@" ; exit $?
 
 //JAVA 21
-//DEPS org.kohsuke:github-api:1.327
+//DEPS org.kohsuke:github-api:1.330
 //DEPS info.picocli:picocli:4.7.7
 
 import org.kohsuke.github.GHIssue;
@@ -195,7 +195,14 @@ class Report implements Runnable {
 			} else {
 				sb.append("Unfortunately, the build failed!\n\n");
 				if (!dryRun) {
-					issue.reopen();
+					try {
+						issue.reopen();
+					} catch (Exception e) {
+						System.out.println("ERROR: Failed to open issue: " + issue.getHtmlUrl());
+						System.out.println("ERROR: Make sure the issue is owned by @mandrel-bot and that it was not closed by another user");
+						e.printStackTrace();
+						System.exit(1);
+					}
 				}
 				System.out.println("The issue has been re-opened");
 			}

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -92,7 +92,7 @@ jobs:
       quarkus-version: "main"
       mandrel-version: "mandrel/25.0"
       jdk: "25/ea"
-      issue-number: "917"
+      issue-number: "946"
       issue-repo: "graalvm/mandrel"
       mandrel-it-issue-number: "379"
       mandrel-packaging-version: "25.0"

